flink-python/pyflink/table/statement_set.py

```diff
@@ -50,6 +50,13 @@ def add_insert_sql(self, stmt: str) -> 'StatementSet':
         """
         self._j_statement_set.addInsertSql(stmt)
         return self
+    
+    def attach_as_datastream(self: StatementSet):
```

Comment:
Could you also update the documentation here: https://nightlies.apache.org/flink/flink-docs-master/docs/dev/table/data_stream_api/#adding-table-api-pipelines-to-datastream-api

Reply:
Definitely